load.php

```diff
@@ -181,13 +231,8 @@ function perflab_load_active_modules() {
 	}
 
 	foreach ( $active_modules as $module ) {
-		// Do not load module if it no longer exists.
-		$module_file = plugin_dir_path( __FILE__ ) . 'modules/' . $module . '/load.php';
-		if ( ! file_exists( $module_file ) ) {
-			continue;
-		}
 
-		require_once $module_file;
+		require_once plugin_dir_path( __FILE__ ) . 'modules/' . $module . '/load.php';
```

Comment:
See my comment further above, maybe we should rather do the checks for whether the module can be loaded in here? Either directly within this function or via a separate `perflab_get_active_and_valid_modules()`?

Reply:
If we check whether the module can be loaded here then we have to check similar in the `perflab_render_generator()` function.

Reply:
@mukeshpanchal27 Good point. However, then let's go with my above suggestion of adding a reusable function `perflab_get_active_and_valid_modules()` and have that logic in there. This way we can leave `perflab_get_active_modules()` untouched and with a single purpose.

Reply:
Perflab_get_active_and_valid_modules() is introduced with changes.